invenio_app_ils/ui/src/invenio_app_ils/authentication/SessionManager.js

```diff
@@ -0,0 +1,76 @@
+import { authenticationService } from './services/AuthenticationService';
```

Comment:
I would have put the `SessionManager` in `/services/` too and remove the `authentication` folder...

Reply:
But I have also the `AuthenticationGuard` component there, but I will move `SessionManager` ;)